From 123f9e43f721a11e1b70c49b05cacb8f6edd7eaf Mon Sep 17 00:00:00 2001
From: Sean McNeil <sean@mcneil.com>
Date: Tue, 13 May 2008 18:54:41 +0100
Subject: [PATCH] minor problem with LCD

Looking at wrong register to see if the LCD is turned off?
---
 drivers/i2c/chips/pcf50633.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/i2c/chips/pcf50633.c b/drivers/i2c/chips/pcf50633.c
index db0589e..e3f0271 100644
--- a/drivers/i2c/chips/pcf50633.c
+++ b/drivers/i2c/chips/pcf50633.c
@@ -1459,7 +1459,7 @@ static int pcf50633bl_get_intensity(struct backlight_device *bd)
 	struct pcf50633_data *pcf = bl_get_data(bd);
 	int intensity = reg_read(pcf, PCF50633_REG_LEDOUT);
 
-	if (!(reg_read(pcf, PCF50633_REG_LEDOUT) & 1))
+	if (!(reg_read(pcf, PCF50633_REG_LEDENA) & 1))
 		intensity = 0;
 
 	return intensity & 0x3f;
@@ -1472,7 +1472,7 @@ static int pcf50633bl_set_intensity(struct backlight_device *bd)
 	int old_intensity = reg_read(pcf, PCF50633_REG_LEDOUT);
 	int ret;
 
-	if (!(reg_read(pcf, PCF50633_REG_LEDOUT) & 1))
+	if (!(reg_read(pcf, PCF50633_REG_LEDENA) & 1))
 		old_intensity = 0;
 
 	if ((bd->props.power != FB_BLANK_UNBLANK) ||
-- 
1.5.6.5

